CompilerSource/compiler/jdi_utility.cpp

```diff
@@ -55,6 +55,11 @@ static void iterate_overloads(
        iter != d->overloads.end(); iter++) {
     visit_overload(iter->second, min, max, varargs_t);
   }
+
+  for (definition_template* templateOverload : d->template_overloads) {
+    if (min > templateOverload->params.size()) min = templateOverload->params.size();
```

Comment:
This is actually the template parameter size; it doesn't actually correspond with the number of parameters for the function. The function will  be declared within the template scope pointed to by `templateOverload`.